mingw-w64-git/git-wrapper.c

```diff
@@ -13,13 +13,16 @@
 #include <shellapi.h>
 #include <stdio.h>
 #include <wchar.h>
+#include <stdbool.h>
```

Comment:
I wonder whether we can do without this...

Reply:
`IsRunningOnARM64()` is a boolean (`bool IsRunningOnARM64()`). Without `stdbool.h` this results in `error: unknown type name 'bool'`.

We could surely change it to a different data type, but isn't a boolean the cleanest solution here? 😊

Reply:
> We could surely change it to a different data type, but isn't a boolean the cleanest solution here? 😊

It is. Thank you for your patient explanation.